django/contrib/admin/checks.py

```diff
@@ -3,11 +3,15 @@
 
 from itertools import chain
 
-from django.contrib.admin.utils import get_fields_from_path, NotRelationField, flatten
+from django.contrib.admin.utils import (
```

Comment:
We discussed this in #django-dev a bit yesterday. My main concern is that it will make it difficult to sort imports manually as people who have a right margin at 80 characters would need to change it just for this. We prefer longer lines "when it helps readability". I don't think there's much difference in readability of imports, so I prefer to keep things simple.